editor.planx.uk/src/@planx/components/NumberInput/model.ts

```diff
@@ -28,6 +29,7 @@ export const parseNumberInput = (
   fn: data?.fn || "",
   units: data?.units,
   allowNegatives: data?.allowNegatives || false,
+  onlyWholeNumbers: data?.onlyWholeNumbers || false,
```

Comment:
Not wedded to this variable naming, happy to receive suggestions

Reply:
Maybe `format: "integer" | "float"` is a little more explicit? Or even `isInteger: boolean`?